mibitracker/request_helpers.py

```diff
@@ -562,7 +563,8 @@ def get_channel_data(self, image_id, channel_name):
             raise MibiTrackerError(
                 'Specified channel name is not present in image')
         buf = io.BytesIO()
-        response = requests.get(image_info['overlays'][channel_name])
+        response = requests.get(image_info['overlays'][channel_name],
+                                timeout=30)
```

Comment:
Let's have module-level constants for the mibitracker timeouts and the external url timeouts.

Reply:
done